modules/localstack/src/test/java/org/testcontainers/containers/localstack/LocalstackContainerTest.java

```diff
@@ -451,4 +455,53 @@ private String runAwsCliAgainstDockerNetworkContainer(String command) throws Exc
             return logs;
         }
     }
+
+    public static class WithEnvironmentVariables {
+
+        @ClassRule
+        public static LocalStackContainer localstack = new LocalStackContainer(LocalstackTestImages.LOCALSTACK_LATEST)
+            .withServices(Service.S3)
```

Comment:
```suggestion
```

Reply:
Is that correct ? I thought we have to set the S3 service

Reply:
No since LocalStack 0.13 IIRC

Reply:
ok removing then now